pkg/yamlmeta/ast.go

```diff
@@ -27,7 +27,10 @@ type Node interface {
 
 	Check() TypeCheck
 
+	UserFriendlyName() string
```

Comment:
```suggestion
	DisplayName() string
```

Played with this together: is `DisplayName` a better name for this method?

Reply:
Done. Thank you.